packages/acceptance-tests/pkg-tests-specs/sources/pnp.test.js

```diff
@@ -1704,6 +1704,8 @@ describe(`Plug'n'Play`, () => {
       await mkdirp(`${path}/foo`);
       await writeFile(`${path}/foo/index.js`, ``);
 
+      await run(`install`);
```

Comment:
Don't think this test was testing the `pnpapi` before
cc @paul-soporan https://github.com/yarnpkg/berry/pull/1563

Reply:
You're right, nice catch! I think I just copied the test above this one and removed the `add` but forgot to add an `install`.